src/frontend/src/pages/FinancePage/ReimbursementRequestPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -0,0 +1,156 @@
+          </Grid>
+          <Grid item sm={6} xs={12}>
+            <VerticalDetailDisplay label="Refund Source" content={`${reimbursementRequest.account}`} />
+          </Grid>
+          <Grid item sm={6} xs={12}>
+            <VerticalDetailDisplay label="Expense Type" content={`${reimbursementRequest.expenseType.name}`} />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            container
+            mt={2}
+            ml={2}
+            sx={{ backgroundColor: totalCostBackgroundColor, borderRadius: '10px' }}
+          >
+            <Grid item xs={6} textAlign={'center'}>
+              <Typography fontSize={50}>Total Cost</Typography>
+            </Grid>
+            <Grid xs={6} sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+              <Typography fontSize={50}>{`$${reimbursementRequest.totalCost}`}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+      </>
+    );
+  };
+
+  const ReimbursementProductsView = () => {
+    const uniqueWbsElementsWithProducts = new Map<String, ReimbursementProduct[]>();
+    reimbursementRequest.reimbursementProducts.forEach((product) => {
+      const wbs = wbsPipe(product.wbsNum);
+      if (uniqueWbsElementsWithProducts.has(wbs)) {
+        const products = uniqueWbsElementsWithProducts.get(wbs);
+        products?.push(product);
+      } else {
+        uniqueWbsElementsWithProducts.set(wbs, [product]);
+      }
+    });
+
+    const keys = [];
+    for (const key of uniqueWbsElementsWithProducts.keys()) {
+      keys.push(key);
+    }
+
+    return (
+      <>
+        <Typography variant="h5">Products</Typography>
+        <Table component={Paper}>
+          <TableHead>
+            <TableRow>
+              <TableCell>WBS Element</TableCell>
+              <TableCell>Products</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {keys.map((key) => {
+              return (
+                <TableRow>
+                  <TableCell>{key}</TableCell>
+                  <TableCell>
+                    {uniqueWbsElementsWithProducts.get(key)?.map((product) => {
+                      return <Chip label={`${product.name} $${product.cost}`} />;
+                    })}
+                  </TableCell>
+                </TableRow>
+              );
+            })}
+          </TableBody>
+        </Table>
+      </>
+    );
+  };
+
+  const ReceiptsView = () => {
+    return (
+      <Box sx={{ maxHeight: `250px`, overflow: 'auto' }}>
+        <Typography variant="h5">Receipts</Typography>
+        {reimbursementRequest.receiptPictures.map((receipt) => {
+          return (
+            <iframe
+              style={{ height: `200px`, width: '50%' }}
+              src={`https://drive.google.com/file/d/${receipt.googleFileId}/preview`}
+              title="ollie"
+            />
```

Comment:
<img width="316" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/9bd481ec-917c-4f76-ad8c-b5cc4a972512">
looks a lil weird when there's no receipts

Reply:
what should we do about that